units/provisioning_setup_host.go

```diff
@@ -841,28 +827,40 @@ func mountLinuxVolume(ctx context.Context, env evergreen.Environment, h *host.Ho
 	if err != nil {
 		return errors.Wrap(err, "can't refresh device info")
 	}
-	err = client.CreateCommand(ctx).Sudo(true).SetInputBytes([]byte(fmt.Sprintf("UUID=%s /%s auto noatime 0 0\n", device.UUID, evergreen.HomeVolumeDir))).Append("tee --append /etc/fstab").Run(ctx)
+	err = client.CreateCommand(ctx).Sudo(true).SetInputBytes([]byte(fmt.Sprintf("UUID=%s %s auto noatime 0 0\n", device.UUID, h.Distro.HomeDir()))).Append("tee --append /etc/fstab").Run(ctx)
```

Comment:
This seems right, but can we split up this command a little bit for readability?

Reply:
done